vpr/src/base/noc_storage.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+ * The NocStorage defines the embedded NoC in the FPGA device.
+ * The NocStorage contains the following information:
+ * - A list of all the routers in the NoC (NocRouter objects)
+ * - A list of all the links in the NoC (NocLink objects)
+ * - A connections list for each router. This is simply a list
```

Comment:
Same idea; most description should go in .h file.

Reply:
Fixed